sdk/sitemanager/Azure.ResourceManager.SiteManager/src/Generated/Extensions/MockableSiteManagerTenantResource.cs

```diff
@@ -69,8 +69,7 @@ public virtual AsyncPageable<EdgeSiteResource> GetEdgeSitesAsync(string serviceg
             Argument.AssertNotNullOrEmpty(servicegroupName, nameof(servicegroupName));
 
             HttpMessage FirstPageRequest(int? pageSizeHint) => SitesByServiceGroupRestClient.CreateListByServiceGroupRequest(servicegroupName);
-            HttpMessage NextPageRequest(int? pageSizeHint, string nextLink) => SitesByServiceGroupRestClient.CreateListByServiceGroupNextPageRequest(nextLink, servicegroupName);
-            return GeneratorPageableHelpers.CreateAsyncPageable(FirstPageRequest, NextPageRequest, e => new EdgeSiteResource(Client, EdgeSiteData.DeserializeEdgeSiteData(e)), SitesByServiceGroupClientDiagnostics, Pipeline, "MockableSiteManagerTenantResource.GetEdgeSites", "value", "nextLink", cancellationToken);
+            return GeneratorPageableHelpers.CreateAsyncPageable(FirstPageRequest, null, e => new EdgeSiteResource(Client, EdgeSiteData.DeserializeEdgeSiteData(e)), SitesByServiceGroupClientDiagnostics, Pipeline, "MockableSiteManagerTenantResource.GetEdgeSites", "value", null, cancellationToken);
```

Comment:
The next page fetching is missing in the implementation for mgmt due to not generate from the latest spec commit.
This should be fixed during the api-version bump.